File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/If-Else/NumberCheck.java
//Finding whether the number entered by the user is Negative,Positive or zero 

import java.util.*;


public class NumberCheck
{
    public static void main(String[] args) 
    {
        Scanner in= new Scanner(System.in);
        
        System.out.println("Enter the number to be checked whether it is Negative ,Positive or Zero ");
        
        int n=in.nextInt();

        if(n>0)
            System.out.println("The number " + n +" is positive");

        else if(n<0)
            System.out.println("The number " + n +" is negative");
        else 
            System.out.println("The number " + n +" is zero");
        
    }
}

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/If-Else/numbercheck.cpp
#include<iostream>
using namespace std;

int main()
{
    int n;
    cout<<"Enter the number to br checked whether it is positive , negative or zero "<<endl;
    cin>>n;

    if(n<0)
    cout<<"The number "<< n << " is negative"<< endl;
    else if(n>0)
    cout<<"The number "<< n << " is positive "<< endl;
    else
    cout<<"The number "<< n << " is zero "<< endl;
   
}
